va/va.h

```diff
@@ -4225,6 +4225,62 @@ VAStatus vaDeriveImage (
     VAImage *image	/* out */
 );
 
+typedef struct _VASurfaceAllocationInfo{
```

Comment:
I think this structure should be aligned with VASurfaceAttribExternalBuffers defined here: https://github.com/intel/libva/blob/95eb8cf469367b532b391042fa0e77ca513ac94e/va/va.h#L1379

Optimally allocation info structure should be just used inside VASurfaceAttribExternalBuffers, but likely we will have a chance to do that only with the next major version change if it will ever happen.

Anyhow, following comments follow alignment idea.

Reply:
yes, cant add additional member in VASurfaceAttribExternalBuffers now. major version change can break ABI .